tests/evm/test_addmod.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+
+from typing import Optional
+from zkevm_specs.evm import (
+    ExecutionState,
+    StepState,
+    verify_steps,
+    Tables,
+    Block,
+    Bytecode,
+    RWDictionary,
+)
+from zkevm_specs.util import rand_fq, RLC
+
+TESTING_DATA = [
+    (6, 4, 10),
+    (6, 7, 10),
+    (6, 7, 0),
+    (0, 0, 0),
+]
```

Comment:
Perhaps with some overflow testing cases will be nice.

Reply:
Updated in https://github.com/privacy-scaling-explorations/zkevm-specs/pull/189/commits/9342cbdc27cbf936416acc34b616947c06c92ad4